### PR TITLE
Create nth Ugly number | DP  #221

### PR DESCRIPTION
**Pull request by om840**
## Assigned Issue Number?
#001

## Language Opted?
C/C++/Python/Java/...

## Number of files added or updated?
1/2/3

## Name of these files?
main folder => sub_folder => ... => ... =>  file.py

example :
Algorithms => Searching => Linear_Search => Linear_Search.py

### Checklist

- [ ] I was assigned for this otherwise my **PR will be rejected**.
- [ ] I've read the contribution guidelines.
- [ ] I've referred the correct issue number.
- [ ] I've filled up this entire template correctly.

:star2: this repository and support.

### CODE REVIEW

**Comment by Amandixit10** [comment]
Hi ! is this repo liable for hacktober fest ?

**Comment by om840** [comment]
yes sir this repo is  liable for hactober fest

**Comment by Amandixit10** [comment]
please assign me for java.

**Review by krishrahul98 — COMMENTED** [review]
Please write an efficient approach and fill PR template properly.

**Comment by krishrahul98** [comment]
> Hi ! is this repo liable for hacktober fest ?

@Amandixit10 Yes, any repo with topic **hacktoberfest** is liable or if your **PR** is labelled with **hacktberfest-accepted** tag.

File: Algorithms/DP/nth Ugly number | DP.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+	//Check for all integers untill  ugly count becomes n 
+
+	while n > count: 
+		i += 1
+		if isUgly(i): 
+			count += 1
+	return i 
+
+
```
Comment (hemanth-kotagiri) [inline Algorithms/DP/nth Ugly number | DP.py:31]: Please format your code properly and also add a ``main`` method to demonstrate the solution.

**Review by hemanth-kotagiri — CHANGES_REQUESTED** [review]
Hello. Here are a few things that you need to change.
1. Always format your code properly. It is essential because once your code is merged into master of the upstream repository, it will be used by many others. And it is also a good practice of writing clean and structured code.
2. Please remove the ``//`` from the following lines. It is not way you do comments in Python. So replace it with the right one.

File: Algorithms/DP/nth Ugly number | DP.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+ 
+
+// This function divides a by greatest divisible power of b
```
[the remark this answers is not in the document]
Comment (krishrahul98) [reply Algorithms/DP/nth Ugly number | DP.py:3]: yes, @om840 please update your code with proper formatting and better time complexity as well.

File: Algorithms/DP/nth Ugly number | DP.py (hunk 1 above)
Comment (krishrahul98) [reply Algorithms/DP/nth Ugly number | DP.py:31]: great suggestion

**Comment by hemanth-kotagiri** [comment]
Hey, @krishrahul98. I would be closing this PR since it has been inactive for almost a year now. @om840 I request you to raise another PR regarding this with better implementation.

**Pull request closed without merging**